src/sentry/models/debugfile.py

```diff
@@ -363,6 +363,24 @@ def _analyze_progard_filename(filename: str) -> Optional[str]:
         return None
 
 
+@region_silo_only_model
+class ProguardArtifactRelease(Model):  # type: ignore
```

Comment:
what's the `# type: ignore` here?  mypy should be fine with subclassing `Model` now

Reply:
Yup should not be there, thx